features/users/user_list.feature

```diff
@@ -2,26 +2,36 @@
 Feature: As a site owner
   So I can make collaboration among registered users easier
   I would like to display a index of users with links to user profiles
+  And allow users to search for other users using a variety of criterias
 
   Background:
     Given I am on the "home" page
-    And the following users exist
-      | first_name | last_name | email                   |
-      | Alice      | Jones     | alice@btinternet.co.uk  |
-      | Bob        | Butcher   | bobb112@hotmail.com     |
-      |            | Croutch   | c.croutch@enterprise.us |
-      | Dave       |           | dave@dixons.me          |
-    And I am logged in as user with email "brett@example.com", with password "12345678"
+    And the following projects exist:
+      | title         | description             | pitch       | status   | github_url                                  | pivotaltracker_url                               |
+      | hello world   | greetings earthlings    |             | active   | https://github.com/AgileVentures/WebsiteOne | https://www.pivotaltracker.com/s/projects/742821 |
+      | hello mars    | greetings aliens        |             | inactive |                                             |                                                  |
+      | hello jupiter | greetings jupiter folks |             | active   |                                             |                                                  |
+      | hello mercury | greetings mercury folks |             | inactive |                                             |                                                  |
+      | hello saturn  | greetings saturn folks  | My pitch... | active   |                                             |                                                  |
+      | hello sun     | greetings sun folks     |             | active   |                                             |                                                  |
+    And there are no videos
```

Comment:
for readability, I would include only the columns that are used in the scenarios.